tasks/restore_engine_from_file.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Run engine cleanup command
+  command: "engine-cleanup"
+  when: ovirt_engine_setup_restore_engine_cleanup | bool
```

Comment:
The filter `bool` is needed if it is a string you can remove it.

Reply:
ovirt_engine_setup_restore_engine_cleanup is bool which defined in the defaults file as false

Reply:
yes but the filter usage is to change `"True"` -> `True` or `"False"` -> `False` etc.
https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html

Reply:
Done